### DISCUSSION TITLE
How to use this

### DISCUSSION THREAD

**Discussion by 6482985729** [Bon-Appetit/porn-domains#8] [Q&A] [answered]
Hey guys,
How do I use this list?
Currently, I'm using freedom to block porn, but you can only paste in 50 websites at a time to a blocklist. That would take forever with this.
Many thanks,
Corey

**Comment by CodeAlDente** [comment, marked as answer]
One way to use the list is to put all the domains into your `hosts` file or even combine it with [Pi-Hole](https://pi-hole.net) to block access to all those domains on your network.
